src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -38,7 +39,8 @@ const options: Partial<Options> = {
         {
             name: 'preventOverflow',
             options: {
-                padding: 5,
+                altAxis: true,
```

Comment:
Added this property to address the case where the user has drawn on all four edges of the document (if we don't have this property, then the toolbar goes up off of the visible portion of the viewport.